OKR.md

```diff
@@ -2,6 +2,32 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q3
+
+**The Connectivity Magic story is figured out and solved**
+
+- A js-ipfs Daemon will always find other IPFS Daemons (js & go) in the LAN 
+- A js-ipfs node (Browser & Desktop) always finds files that are on the Gateways
+- The rendezvous protocol is a sound solution for DApps (e.g. PeerPad & others)
+- The DHT is enabled by default in js-ipfs
+- Handles correctly a scenario where the number of Connections grows (through Conn Management Strategy)
```

Comment:
In this KR, the implicit things are:

- js-ipfs should have a good Conn Mgmt defaults
- Conn Mgmt should be part of the docs of js-ipfs
- There should be instructions (i.e. Tutorial) on how to apply Conn Mgmt
- This needs to be severely tested
- No browser shall crash from too many connections.